pkg/scalers/datadog_scaler.go

```diff
@@ -191,7 +191,7 @@ func newDatadogConnection(ctx context.Context, meta *datadogMetadata, config *Sc
 	configuration.HTTPClient = kedautil.CreateHTTPClient(config.GlobalHTTPTimeout, false)
 	apiClient := datadog.NewAPIClient(configuration)
 
-	_, _, err := apiClient.AuthenticationApi.Validate(ctx) //nolint:bodyclose
```

Comment:
Please, don't do this, the CI fails

Reply:
Yes, that is what I was referring to. For me the linter was failing with the line as I was on earlier version.